cypress/support/pages/home.pageObject.js

```diff
@@ -7,9 +7,16 @@ class HomePageObject extends PageObject {
     return cy.getByDataCy('username-link');
   }
 
-  assertHeaderContainUsername(username) {
-    this.usernameLink
-      .should('contain', username);
+  assertHeaderContainUsername = (username) => {
+    cy.get('[data-cy="username-link"]').should('contain', username);
+  };
+
+  clickNewArticle() {
```

Comment:
It's better to find elements by data-qa attributes instead of text, for the sake of accessibility and to avoid potential localization issues.

Reply:
I had wrote at Slack-chat about problems with data-qa attributes at my local machine
[https://github.com/mate-academy/qa_cypress_e2e_project/pull/url](url)
![e2e_12](https://github.com/mate-academy/qa_cypress_e2e_project/assets/13405550/c7a4a14f-653e-4024-b742-3d028711420d)